pkg/occlient/occlient_test.go

```diff
@@ -1201,7 +1201,7 @@ func TestCleanupAfterSupervisor(t *testing.T) {
 				}
 
 				if lenVolumes-1 != len(updatedDc.Spec.Template.Spec.Volumes) {
-					t.Errorf("wrong number of Volumes deleted, expected: %s, removed %s", "1 volume", lenVolumes-len(updatedDc.Spec.Template.Spec.Volumes))
+					t.Errorf("wrong number of Volumes deleted, expected: %s, removed %v", "1 volume", lenVolumes-len(updatedDc.Spec.Template.Spec.Volumes))
```

Comment:
here %d can also be used since the value of lenVolumes-len(updatedDc.Spec.Template.Spec.Volumes) is an int

Reply:
Done and fixed